src/webview/static/base.css

```diff
@@ -142,14 +142,18 @@ body {
   padding: 0 16px 16px 80px;
 }
 
-.message-muted {
+.message[data-mute-state="hidden"] {
```

Comment:
Then the rest of these first four commits (i.e. everything in those commits apart from adding the `_` plumbing) would be best squashed together. That will simplify the changes quite a bit, because there's a lot of bits like this one that do one thing in an earlier commit and then go a different way in a later commit.

Which is a very natural and useful thing to do while drafting how something should work! Just when it's done, it's quite helpful to clean up the series so it's optimized for the reader to understand.

Reply:
Done! I sort of assumed that the whole diff would be larger than I wanted, but looking back at it it's actually pretty simple, which is nice.

Reply:
Yeah :smile: 

There's actually one further squashing I had in mind relative to your new revision, which is:

`$ git uvr --shortstat`
3dc5af683 webview: Add initial support for hiding muted messages.
 4 files changed, 63 insertions(+), 2 deletions(-)
229ddf0d5 webview [nfc]: Pass GetText into messageAsHtml.
 4 files changed, 14 insertions(+), 3 deletions(-)
a1e07759e messageAsHtml: Translate user-muted message.
 2 files changed, 5 insertions(+), 5 deletions(-)

to squash those first and third changes (therefore coming after the "pass GetText in" change). The resulting change is
 5 files changed, 63 insertions(+), 2 deletions(-)

so actually not any bigger than the version that doesn't translate the text, and it cleans the history up further.

Everything else looks good, so I'll just merge now and make that change on the way.